packages/ag-charts-enterprise/src/features/zoom/zoom.ts

```diff
@@ -454,7 +457,20 @@ export class Zoom extends _ModuleSupport.BaseModuleInstance implements _ModuleSu
 
         if (!seriesRect) return;
 
-        event.preventDefault();
+        const zoom = this.getZoom();
+        const isZoomCapped = (this.isMaxZoom(zoom) && event.deltaY > 0) || (this.isMinZoom(zoom) && event.deltaY < 0);
+
+        if (!this.isFirstWheelEvent || !isZoomCapped) {
+            event.preventDefault();
+        }
+
+        // Prevent browser scrolling when smooth wheel events continue being fired after the chart
+        // reaches a min or max extent
+        this.isFirstWheelEvent = false;
+        if (this.wheelTimer != null) clearTimeout(this.wheelTimer);
+        this.wheelTimer = setTimeout(() => {
+            this.isFirstWheelEvent = true;
+        }, 100);
```

Comment:
Nit: I don't like these inline `setTimeout()`s, as it can make testing tricky -  maybe we could use a utility function like `debouncedCallback()` to create a debounce queue/processor to clean this up a bit?